src/Microsoft.AspNet.Server.Kestrel/ServerFactory.cs

```diff
@@ -43,7 +43,8 @@ public IDisposable Start(IServerInformation serverInformation, Func<IFeatureColl
             {
                 disposables.Add(engine.CreateServer(
                     address.Scheme,
-                    address.Host,
+                    // Unix sockets use a file path, not a hostname.
+                    address.Scheme == Constants.UnixScheme ? address.Path : address.Host,
```

Comment:
No, but we don't use the System.Uri parser for any of this. We have to support wildcards anyways.